celery/backends/base.py

```diff
@@ -73,6 +72,19 @@ def unpickle_backend(cls, args, kwargs):
     return cls(*args, app=current_app._get_current_object(), **kwargs)
 
 
+
+if sys.version_info < (3, 0, 0):
+    from six import reraise as _reraise
```

Comment:
is it available in celery.five?

Reply:
ah yes, absolutely: https://github.com/celery/vine/blob/master/vine/five.py#L192

Reply:
and you don't need any of this `if sys.version_info` !